src/Examine.AzureDirectory/AzureDirectoryFactory.cs

```diff
@@ -46,6 +52,10 @@ public AzureDirectoryFactory(bool isReadOnly)
         /// The <see cref="Lucene.Net.Store.Directory"/>.
         /// </returns>
         public override Lucene.Net.Store.Directory CreateDirectory(DirectoryInfo luceneIndexFolder)
+        {
+           return CreateDirectory(luceneIndexFolder, _loggingService);
+        }
+        public Directory CreateDirectory(DirectoryInfo luceneIndexFolder, ILoggingService loggingService)
```

Comment:
Why do we need this method?

Reply:
I think it is left over after previous implementation, where I was using that interface, will clean up :)